.teamcity/src/main/kotlin/common/VersionedSettingsBranch.kt

```diff
@@ -2,7 +2,28 @@ package common
 
 import jetbrains.buildServer.configs.kotlin.v2019_2.DslContext
 
-data class VersionedSettingsBranch(val branchName: String, val enableTriggers: Boolean) {
+data class VersionedSettingsBranch(val branchName: String) {
+    /**
+     * Whether the <a href="https://www.jetbrains.com/help/teamcity/configuring-vcs-triggers.html">VCS trigger</a>
+     * should be enabled, i.e. when new commits are pushed to this branch, should a ReadyForNightly job
+     * be triggered automatically?
+     *
+     * Currently, we only enable VCS trigger for `master` and `release` branches.
+     */
+    val enableVcsTriggers: Boolean = mainBranches.contains(branchName.lowercase())
```

Comment:
❓ Why shouldn't we enable VCS triggers on the `release...` branches? I suppose we shouldn't enable VCS triggers on the `experimental` branch, though for the old release branches it seems to make sense to run a build when there are changes.

Reply:
That's how the code works before. But yes, it makes sense to run them for old release branches.